e2e/tests/functional/plugins/plot/stackedPlot.e2e.spec.js

```diff
@@ -228,6 +229,46 @@ test.describe('Stacked Plot', () => {
     ).toContainText(swgC.name);
   });
 
+  test.only('Changing properties of an immutable child plot are applied correctly', async ({
+    page
+  }) => {
+    await page.goto(stackedPlot.url);
+
+    // Go into edit mode
+    await page.getByLabel('Edit Object').click();
+
+    await page.getByRole('tab', { name: 'Config' }).click();
+
+    // Click on canvas for the 1st plot
+    await page.locator(`[aria-label="Stacked Plot Item ${swgA.name}"]`).click();
+
+    // Expand config for the series
+    await page.locator('li.c-tree__item.menus-to-left .c-disclosure-triangle').click();
+
+    // turn off alarm markers
+    await turnOffAlarmMarkers(page);
+
+    // save
+    await page.click('button[title="Save"]');
+    await page.getByRole('listitem', { name: 'Save and Finish Editing' }).click();
+
+    // reload page
+    await page.reload();
+
+    // Click on the 1st plot
+    await page.locator(`[aria-label="Stacked Plot Item ${swgA.name}"] canvas`).nth(1).click();
+
+    await page.locator('li.c-tree__item.menus-to-left .c-disclosure-triangle').click();
+
+    // Assert that alarm markers are still turned off
+    await expect(
+      await page
+        .getByRole('list', { name: 'Plot Series Properties' })
+        .locator('[title="Alarm Markers"]>div')
+        .nth(1)
+    ).toContainText('Disabled');
```

Comment:
We should find a better way of locating this element without using the locator method, nth, or title locators.

Ideally, this assertion should look like this:
```js
await expect(await page.getByRole('listitem', { name: object.name }).getByLabel('Alarm Marker's).toContainText('Disabled')